config/locales/simple_form.fr.yml

```diff
@@ -3,34 +3,34 @@ fr:
   simple_form:
     hints:
       defaults:
-        autofollow: Les personnes qui s'inscrivent grâce à l'invitation vous suivront automatiquement
+        autofollow: Les personnes qui s’inscrivent grâce à l’invitation vous suivront automatiquement
         avatar: Au format PNG, GIF ou JPG. 2 Mo maximum. Sera réduit à %{dimensions}px
         bot: Ce compte exécute principalement des actions automatisées et pourrait ne pas être surveillé
-        context: Un ou plusieurs contextes où le filtre devrait s'appliquer
+        context: Un ou plusieurs contextes où le filtre devrait s’appliquer
         digest: Uniquement envoyé après une longue période d’inactivité et uniquement si vous avez reçu des messages personnels pendant votre absence
         display_name:
           one: <span class="name-counter">1</span> caractère restant
           other: <span class="name-counter">%{count}</span> caractères restants
-        fields: Vous pouvez avoir jusqu'à 4 éléments affichés en tant que tableau sur votre profil
+        fields: Vous pouvez avoir jusqu’à 4 éléments affichés en tant que tableau sur votre profil
         header: Au format PNG, GIF ou JPG. 2 Mo maximum. Sera réduit à %{dimensions}px
-        inbox_url: Copiez l'URL depuis la page d'accueil du relais que vous souhaitez utiliser
+        inbox_url: Copiez l’URL depuis la page d’accueil du relais que vous souhaitez utiliser
         irreversible: Les pouets filtrés disparaîtront irrémédiablement, même si le filtre est supprimé plus tard
-        locale: La langue de l'interface-utilisateur, des courriels, et des notifications
+        locale: La langue de l’interface utilisateur·ice, des courriels, et des notifications
```

Comment:
remove the last `,`

Reply:
I wouldn't do so.

Reply:
je suis pour la virgule d’Oxford également, si on l’utilise avec consistance partout